.github/workflows/test-ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Test CI
+
+on: [push, pull_request]
+
+jobs:
+  auto-testing:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 10
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v3
+      with:
+        python-version: 3.8
+    - name: Add conda to system path
+      run: |
+        # $CONDA is an environment variable pointing to the root of the miniconda directory
+        echo $CONDA/bin >> $GITHUB_PATH
+    - name: Install conda env & dependencies
+      run: |
+       conda install pytorch torchvision torchaudio cudatoolkit=11.3 -c pytorch
+       pip install -e .
+       conda list
+    - name: Run tests
+      run: |
+        ./all_tests.sh
+#    - name: Install codecov dependencies
```

Comment:
Why are these commented out?

Reply:
Well, if we open it, you can refer to this link : https://github.com/wmFrank/sample-factory/runs/7034191959?check_suite_focus=true
It does show the coverage situation:
<img width="759" alt="image" src="https://user-images.githubusercontent.com/30235642/175505119-4b2f2c58-9b15-4487-b52d-c128baef4e14.png">
but it also checks all the test in the project:
<img width="967" alt="image" src="https://user-images.githubusercontent.com/30235642/175505409-c59dfc78-4f1d-4948-858c-c316e69ed3cc.png">
And you can see below, it blocks the upload of the coverage report.
So we won't see any thing like this:
<img width="1312" alt="image" src="https://user-images.githubusercontent.com/30235642/175506064-60454096-3a32-4687-99c9-a8cda62173b0.png">